docs/Hosts/Spawn-Hosts.md

```diff
@@ -55,6 +55,186 @@ lifetime up to 30 days past host creation.
 If you'd like to get a notification before a host expires, you can [set up a
 notification](../Project-Configuration/Notifications#spawn-host-expiration) for it.
 
+## Unexpirable Host Sleep Schedules
+
+Evergreen has introduced a new feature for unexpirable hosts called host sleep schedules, which allow you to control
+when you'd like your unexpirable host to be automatically turned on or off. A sleep schedule lets you choose recurring
+times of the week when you want your host to be on, and when it's okay for it to be turned off. For example, you could
+set a sleep schedule where your host is up and running during your work hours from 10 am to 6 pm between Monday and
+Friday and otherwise let it turn it off overnight and during weekends. Setting a sleep schedule ensures your host is on
+during your working hours, while also stopping the host when it's not being actively used. Powering down hosts when
+they're not being used is important to ensure that hosts are being efficiently utilized and to avoid over-spending on
+idle long-lived hosts.
+
+**While this feature is being rolled out, please [choose and complete one of the three options](#beta-testing) for your
+unexpirable host(s). You must complete one of the options before the beta test ends at 11 am (Eastern Time) on Tuesday
+September 3. If you do not take any action, a default schedule will be set on your behalf for your unexpirable host(s)
+at the end of the beta test period.**
+
+### ***Important Note***: This Feature is in Beta Testing! {#beta-testing}
+
+<!-- TODO (DEVPROD-4055): do not merge until comms are ready and a timeline for beta/full rollout is chosen.  -->
```

Comment:
Should this be removed?

Reply:
Good catch and I actually missed filling in one of the dates, should be good now.